Style upgrade for PuppyPaste

jorgediazgutierrez opened the pull request.

jorgediazgutierrez committed:
Added FontAwesome

jorgediazgutierrez committed:
Footer changes

jorgediazgutierrez committed:
Added Clipboard.js Library

jorgediazgutierrez committed:
Changed Logo background

jorgediazgutierrez committed:
Initialized Clipboard.js instance

jorgediazgutierrez committed:
Major Visual Changes on Puppy Paste Layout

jorgediazgutierrez committed:
Relocated Font-Awesome inside /lib folder.

jorgediazgutierrez committed:
Formatting Fix all across the HTML file

jorgediazgutierrez committed:
Header Menu links sent to _blank windows

dergachev commented:
Great stuff! Some improvements:
- copy-paste icon improvement, or at least acknowledgements when copied
- less gray text in output box
- rounded corners are inconsistent
- restore description under message

![](https://dl.dropbox.com/u/29440342/screenshots/RWFFWQEZ-2016.02.01-13-59-00.png)

jorgediazgutierrez committed:
Latest Changes

- copy-paste icon improvement, or at least acknowledgements when copied
- less gray text in output box
- rounded corners are inconsistent
- restore description under message

jorgediazgutierrez committed:
Fixed unnecesary JS change

jorgediazgutierrez commented:
Done!
